packages/component-library/src/InitialFocus.ts

```diff
@@ -32,5 +50,13 @@ export function InitialFocus({ children }: InitialFocusProps) {
   if (typeof children === 'function') {
     return children(node);
   }
-  return cloneElement(children, { inputRef: node });
+
+  const child = Children.only(children);
+  if (isValidElement(child)) {
+    // @ts-ignore `ref` is not a valid prop for the type that `cloneElement` expects, but this feature doesn't work without it.
```

Comment:
This comment can now be removed
```suggestion
    // @ts-ignore `ref` is not a valid prop for the type that `cloneElement` expects, but this feature doesn't work without it.
```